webpack/components/meetup.js

```diff
@@ -16,13 +16,18 @@ export function getNextMeetup(callback) {
 
 export function renderNextMeetup(meetupData) {
   const nextMeetup = meetupData[0];
-  const nextMeetupDate = new Date(nextMeetup.local_date+"T"+nextMeetup.local_time);
```

Comment:
`time` from Meetup is the UTC start time of the event, in milliseconds since the epoch. We don't have to attempt reconstruction from a string.